src/server/api/routers/team/types.ts

```diff
@@ -0,0 +1,5 @@
+import {type joinUserTeamSchema, type leaveUserTeamSchema} from "~/server/api/routers/team/constants";
```

Comment:
For future reference, you can factor out type like this:

`import type {joinUserTeamSchema, leaveUserTeamSchema} from ....`

No need to go in and change everything. This can be automated with linters.